gravitee-apim-console-webui/src/management/settings/settings.module.ts

```diff
@@ -59,6 +81,31 @@ import { DocumentationModule } from '../../components/documentation/documentatio
     PortalSettingsModule,
     ApiPortalHeaderModule,
     CategoriesModule,
+    GioFormSlideToggleModule,
+    GioPermissionModule,
+    MatButton,
+    MatCard,
+    MatCardContent,
+    MatCardHeader,
+    MatCardSubtitle,
+    MatCardTitle,
+    MatCell,
+    MatCellDef,
+    MatColumnDef,
+    MatHeaderCell,
+    MatHeaderRow,
+    MatHeaderRowDef,
```

Comment:
maybe MatTableModule should be relevant not ? 

I often try to group imports when possible to avoid too many unnecessary lines here 🤷‍♂️